lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -459,6 +459,13 @@ inventory:
     config_get: "show inventory"
     config_get_token: ["TABLE_inv", "ROW_inv", 0, "vendorid"]
 
+ntp_config:
+  config:
+    config_get: "show running-config | include 'ntp source-interface'"
```

Comment:
Mike, see my comment about using 'show running-config ntp all' instead of this pipe.